src/routes/trino.js

```
@@ -135,6 +135,86 @@ router.get("/v1/statement/:state/:queryId/:keyId/:num", async (req, res) => {
       return res.status(404).json({ error: "Query not found" });
     }
 
+    if (query.status === QUERY_STATUS.NO_VALID_CLUSTERS){
+        logger.warn("No valid clusters", { state, queryId });
+
+        // Create new error return to tell Trino client the query failed
+
+        let response = {
```

Why not just return a 400 error code with a message? Since the query never actually makes it to a Trino cluster for execution, I think this error gives the wrong idea that Trino failed the query.

If this body is necessary for something else, can this be pulled out into a helper function / file so that it doesn't pollute the implementation?